server/sonar-db-core/src/main/resources/org/sonar/db/version/schema-h2.ddl

```diff
@@ -610,11 +610,18 @@ CREATE TABLE "CE_ACTIVITY" (
   "ERROR_MESSAGE" VARCHAR(1000),
   "ERROR_STACKTRACE" CLOB(2147483647)
 );
+
 CREATE UNIQUE INDEX "CE_ACTIVITY_UUID" ON "CE_ACTIVITY" ("UUID");
 CREATE INDEX "CE_ACTIVITY_COMPONENT_UUID" ON "CE_ACTIVITY" ("COMPONENT_UUID");
 CREATE INDEX "CE_ACTIVITY_ISLASTKEY" ON "CE_ACTIVITY" ("IS_LAST_KEY");
 CREATE INDEX "CE_ACTIVITY_ISLAST_STATUS" ON "CE_ACTIVITY" ("IS_LAST", "STATUS");
 
+CREATE TABLE "CE_TASK_CHARACTERISTICS" (
+  "UUID" VARCHAR(40) NOT NULL PRIMARY KEY,
+  "TASK_UUID" VARCHAR(40) NOT NULL,
```

Comment:
An index is missing on TASK_UUID in order to quickly load the characteristics of a task (see the SQL request `selectTaskCharacteristics`).